app/views/admin/videos/new.html.haml

```diff
@@ -0,0 +1,23 @@
+%section.admin_add_video
+  .container
+    .row
+      .col-md-10.col-md-offset-1
+        = bootstrap_form_for [:admin, @video], layout: :horizontal, label_col: "col-sm-3", control_col: "col-sm-3" do |f|
+          %ul.nav.nav-tabs
+            %li
+              %a(href="/ui/admin_views_payments") Recent Payments
+            %li.active
+              %a(href="") Add a New Video
+          %br
+          %fieldset
+            = f.text_field :title, label: "Title"
+            = f.select :category_id, options_for_select(Category.all.map {|category| [category.name, category.id]}), control_col: "col-sm-6"
+            = f.text_area :description, rows: 8, label_class: "col-sm-3", control_col: "col-sm-6"
+            = f.file_field :large_cover, control_col: "col-sm-6", label: "Large Cover", class: "btn btn-file"
```

Comment:
I see you are setting `control_col: "col-sm-6"` here and in below fields, instead I would delete this setting inline and set it correctly on line 5 in `bootstrap_form_for`. That will adjust `control_col: ` for all fields. Same with `label_class: "col-sm-3"` - same purpose has `label_col`.